clock_test.go

```diff
@@ -32,6 +32,9 @@ import (
 type constantClock time.Time
 
 func (c constantClock) Now() time.Time { return time.Time(c) }
+func (c constantClock) NewTicker(d time.Duration) *time.Ticker {
+	return time.NewTicker(d)
```

Comment:
Behaviorally, I would assume that a constantClock will never advance the ticker since time never moves.